packages/p2p/src/components/modal-manager/modals/order-details-confirm-modal/order-details-confirm-modal.jsx

```diff
@@ -31,81 +46,68 @@ const OrderDetailsConfirmModal = () => {
-                        defaultChecked={is_checkbox_checked}
-                        label={
-                            is_buy_order_for_user ? (
-                                <Localize i18n_default_text="Yes, I've paid" />
-                            ) : (
-                                <Localize
-                                    i18n_default_text="I've received {{amount}} {{currency}}"
-                                    values={{
-                                        amount: Number(roundOffDecimal(amount * rounded_rate)).toFixed(2),
-                                        currency: local_currency,
-                                    }}
-                                />
-                            )
-                        }
+                    <FileUploaderComponent
+                        accept={'image/png, image/jpeg, image/jpg, application/pdf'}
+                        filename_limit={26}
+                        hover_message={localize('Upload receipt here')}
+                        max_size={max_pot_file_size}
+                        multiple={false}
+                        onDropAccepted={handleAcceptedFiles}
+                        onDropRejected={handleRejectedFiles}
+                        validation_error_message={document_file.error_message}
+                        value={document_file.files}
+                        onClickClose={removeFile}
+                        upload_message={localize('Upload receipt here')}
                     />
                 </Modal.Body>
-                <Modal.Footer>
+                <Modal.Footer className='order-details-confirm-modal__footer'>
                     {order_details_store.error_message && <FormError message={order_details_store.error_message} />}
                     <Button.Group>
                         <Button secondary type='button' onClick={hideModal} large>
-                            {is_buy_order_for_user ? (
-                                <Localize i18n_default_text="I haven't paid yet" />
-                            ) : (
-                                <Localize i18n_default_text='Cancel' />
-                            )}
+                            <Localize i18n_default_text='Go Back' />
                         </Button>
                         <Button
-                            is_disabled={!is_checkbox_checked}
+                            is_disabled={document_file.files?.length === 0}
                             primary
                             large
                             onClick={() => {
+                                sendbird_store.sendFile(document_file.files[0]);
                                 hideModal();
-                                setIsCheckboxChecked(false);
-                                order_store.confirmOrderRequest(id, is_buy_order_for_user);
+                                order_store.confirmOrderRequest(id, true);
```

Comment:
why do we pass in true, as last time we checked if it was a buy order for the user?

Reply:
We needed to send the is_buy_order_for_user parameter when we were using the same modal for sell order and buy order. But right now, we use this only for buy order. So since now we are not using the param anymore, we can directly pass the value here which will always be true.